atomate/vasp/workflows/base/lobster.py

```diff
@@ -0,0 +1,227 @@
+        user_supplied_basis (dict): dict to supply basis functions for each element type
+        isym (int): isym setting during the vasp calculation, currently lobster can only deal with isym=-1
+        c (dict): configurations dict which can include "VASP_CMD", "LOBSTER_CMD", "DB_FILE"
+        additional_outputs (list): list of additional files to be stored in the
+            results DB. They will be stored as files in gridfs. Examples are:
+            "ICOHPLIST.lobster" or "DOSCAR.lobster". Note that the file name
+            should be given with the full name and the correct capitalization.
+
+    Returns: Workflow
+
+
+    """
+
+    c = c or {}
+
+    vasp_cmd = c.get("VASP_CMD", VASP_CMD)
+    lobster_cmd = c.get("LOBSTER_CMD", LOBSTER_CMD)
+    db_file = c.get("DB_FILE", DB_FILE)
+
+    fws = []
+    staticfw = StaticFW(
+        structure=structure,
+        vasp_input_set=LobsterSet(
+            structure,
+            user_incar_settings=user_incar_settings,
+            user_kpoints_settings=user_kpoints_settings,
+            isym=isym,
+        ),
+        vasp_cmd=vasp_cmd,
+        db_file=db_file,
+    )
+    fws.append(staticfw)
+    fws.append(
+        LobsterFW(
+            structure=structure,
+            parents=staticfw,
+            calculation_type=calculation_type,
+            delete_wavecar=delete_all_wavecars,
+            delete_wavecar_previous_fw=delete_all_wavecars,
+            lobster_cmd=lobster_cmd,
+            db_file=db_file,
+            lobsterin_key_dict=user_lobsterin_settings,
+            user_supplied_basis=user_supplied_basis,
+            handler_group="default",
+            validator_group="strict",
+            additional_outputs=additional_outputs,
+        )
+    )
+
+    workflow = Workflow(fws, name="LobsterWorkflow")
+    return workflow
+
+
+def get_wf_lobster_test_basis(
+    structure: Structure,
+    calculation_type: str = "standard",
+    delete_all_wavecars: bool = True,
+    c: dict = None,
+    address_max_basis: Optional[str] = None,
+    address_min_basis: Optional[str] = None,
+    user_lobsterin_settings: dict = None,
+    user_incar_settings: dict = None,
+    user_kpoints_settings: dict = None,
+    isym: int = 0,
+    additional_outputs: List[str] = None,
+) -> Workflow:
+    """
+    creates workflow where all possible basis functions for one compound are tested
+    at the end, the user has to decide which projection worked best (e.g., based on chargespilling)
+    this is the recommended workflow at the moment!
+    Args:
+        structure (Structure): structure object that will be used during the run
+        calculation_type (str): only "standard" is implemented so far
+        delete_all_wavecars (bool): all wavecars wil be deleted if True
+        c (dict): specifications for wf, e.g. VASP_CMD, LOBSTER_CMD etc.
+        address_max_basis (str): address to yaml file including maximum basis set (otherwise predefined file)

```

Comment:
Can I just check if these bases files available somewhere? If so could you point to where they can be found?